docs/DeveloperGuide.md

```diff
@@ -135,14 +135,16 @@ the user's input into key-value pairs, where the keys are specified using `Argum
   leading/trailing whitespace from text, verifying that there are no 
   duplicate entries in the text, and so on.
 
-The activity diagram below describes the workflow of `AddressBookParser`
+The activity diagram below describes the workflow of `NetworkBookParser`
 when determining which `Parser` to use:
-* `TO BE IMPLEMENTED IN 1.4`
+
+<img src="images/AddressBookParser.png" width="1200"/>
 
 The sequence diagram below illustrates the interactions within the
 `FilterCommand` class to generate a `FilterCommand` object, using
 `ArgumentMultiMap` and `ArgumentTokeniser`:
-* `TO BE IMPLEMENTED IN 1.4`
+
+<img src="images/FilterCommandParser.png" width="1200"/>
```

Comment:
I'm not sure if this is necessary, as we have the section on individual command implementation anyway. You can take a look at my PR #297 and improve on it. This diagram can therefore replace my simpler diagram in the PR.

Reply:
If we wanna do that we should do it after merging the two PRs; I'll move the diagram to your part then write a new diagram to demonstrate the workflow of FilterCommandParser finding the correct parse command.

Reply:
Since the simplified diagram has two ref blocks, the related sequence diagrams can be placed in my section of the DG and the FilterCommandParser diagram can go into your section.